wait/http.go

```diff
@@ -184,12 +185,21 @@ func (ws *HTTPStrategy) WaitUntilReady(ctx context.Context, target StrategyTarge
 	address := net.JoinHostPort(ipAddress, strconv.Itoa(port.Int()))
 	endpoint := fmt.Sprintf("%s://%s%s", proto, address, ws.Path)
 
+	// cache the body into a byte-slice so that it can be iterated over multiple times
+	var body []byte
+	if ws.Body != nil {
+		body, err = io.ReadAll(ws.Body)
```

Comment:
I think `io.ReadAll` was added in Go 1.16, right? https://pkg.go.dev/io/ioutil#ReadAll

Reply:
Good catch. I have pushed the change.